launcher/src/main/java/com/threerings/getdown/launcher/ProxyUtil.java

```diff
@@ -96,7 +123,28 @@ public static boolean autoDetectProxy (Application app)
         initProxy(app, host, port, null, null);
         return true;
     }
-
+    
+    
+    
+    private static String findProxyForURL(String upac, Object... args) {
+    	ScriptEngineManager manager = new ScriptEngineManager();
+    	ScriptEngine engine = manager.getEngineByName("javascript");
```

Comment:
Here using openjdk11 I only get back an not-null instance when using javascript with capitals: 
`ScriptEngine engine = manager.getEngineByName("JavaScript");`